packages/ui/src/lib/alert/ErrorMessage.svelte

```diff
@@ -1,16 +1,30 @@
 <script lang="ts">
 import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+import { onMount } from 'svelte';
 import Fa from 'svelte-fa';
 
 import Tooltip from '../tooltip/Tooltip.svelte';
 
 export let error: string;
 export let icon = false;
+export let actionErrorInfo = false;
```

Comment:
I'm wondering if it's a property name that can be understood by callers. The name makes me feel it could be a string to provide while it's expecting a boolean.

maybe should it contain something like what it really does  like `showFullMessage` 

also, should it be the default behavior ?

Reply:
@benoitf `ErrorMessage` is used in many places, so I didn't want to change the original behavior. Those that I changed are the ones we know that have problematic placement with the error message. As we encounter other problematic places similar to the ones fixed here, we can fix them by using that boolean value, and if we notice that most places that use `ErrorMessage` need that fix, we can make it the default behavior. (I can also try and go through all `ErrorMessage` instances and check if this fix could be applied to their placements)
Regarding the property name, I can change it to `wrapErrorInfo`  or `wrapMessage` since the main change is that the text is wrapped across multiple lines instead of showing in one long line

Reply:
`wrapMessage` looks more intuitive for a boolean property

Reply:
leave it as a non-default behavior for now?

Reply:
yes